src/Services/GTT/shared/GTT.Domain/Entities/Challenge.cs

```diff
@@ -8,6 +8,22 @@ namespace GTT.Domain.Entities
 {
     public class Challenge
     {
-        public string Name { get; set; }
+        public int challengeID { get; set; }
```

Comment:
Model is not rule camel, you should capitalize the first letter, the following letters are capitalized for each character
Example: ChallengeId, ClassId